contribute/deprecation-policy.md

```diff
@@ -0,0 +1,34 @@
+# Deprecation policy
+
+Prior to Grafana 10, major releases were an opportunity to make breaking changes and remove code we didn’t want to maintain.
+
+Going forward that is something we will avoid in order to make it easier to upgrade Grafana. From now on we should avoid making unexpected breaking changes to the customer experience and our ability to operate our service.
```

Comment:
```suggestion
Going forward we will do our best to **not** make breaking changes, in order to make upgrading Grafana as easy and reliable as possible. From now on we will avoid making unexpected breaking changes to the customer experience and our ability to operate our service.
```

Reply:
kind of passive voice 🤔 
yours is better but I might take another stab at this one